# base namespace 
string name
---
# true on success 
bool success
